source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -157,6 +179,52 @@ content: |
            npm package. See the `npmjs documentation on the uuid-base64
            package <https://www.npmjs.com/package/uuid-base64>`_ for
            installation instructions.
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 13,14,20,30
+
+           from pymongo import MongoClient
+           from pymongo.encryption_options import AutoEncryptionOpts
+           from pymongo.encryption import ClientEncryption
+           import base64
+           from bson.codec_options import CodecOptions
+           from bson.binary import STANDARD
+           import uuid
+
+           connection_string = "mongodb://localhost:27017"
+           key_vault_namespace = "encryption.__keyVault"
+
+           fle_opts = AutoEncryptionOpts(
+               kms_providers,  # pass in the kms_providers variable from  the previous step
+               key_vault_namespace
+           )
+           client = MongoClient(
+               connection_string,
+               auto_encryption_opts=fle_opts
+           )
+           client_encryption = ClientEncryption(
+               kms_providers,  # pass in the kms_providers variable from  the previous step
+               key_vault_namespace,
+               client,
+               CodecOptions(uuid_representation=STANDARD)
+           )
+
+
+           def create_data_encryption_key():
+
+               data_key = client_encryption.create_data_key("local")
+               uuid_data_key_id = uuid.UUID(bytes=data_key)
+               base_64_data_key_id = base64.b64encode(data_key)
+
+               print("DataKeyId [UUID]: ", str(uuid_data_key_id))
+               print("DataKeyId [base64]: ", base_64_data_key_id)
+
+               return data_key
+
+           data_key = create_data_encryption_key()
```

Comment:
Since anything after this line is out of scope, would it make more sense to call ```create_data_encryption_key()``` without a variable assignment?

Reply:
`data_key` is outside of the function and is accessible if the reader copies and pastes this, it's also required for the next snippet (which mentions using data_key)